iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -26,7 +26,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.38.1-preview-001</Version>
+    <Version>1.38.2-preview-001</Version>
```

Comment:
Can you elaborate on versioning reasoning? Why bump patch version for deviceClient and preview version for other packages?

Reply:
Sure. 
Taking the preview version `1.10.2-preview-003` for example:
Based on our .NET versioning guidelines, we bump the patch version (`2` in 1.10.**2**) in preview when we have API changes that are specific to the preview code. 
When we have preview specific code changes that don't affect the public API surface, or we have changes that are only `main` to `preview` changes, we then bump the preview part of the versioning. (the `-003` part).

Since we updated the preview specific API surface for Microsoft.Azure.Devices.Client, we bumped the patch version there.
Since we have only non-public API surface changes and `main` to `preview` changes in the other packages, we bumped the `-x` preview versioning there.